3.10/aql/operations-replace.md

```diff
@@ -25,104 +25,171 @@ REPLACE <em>keyExpression</em> WITH <em>document</em> IN <em>collection</em></co
 Both variants can optionally end with an `OPTIONS { … }` clause.
 
 `collection` must contain the name of the collection in which the documents should
-be replaced. `document` is the replacement document. When using the first syntax, `document` 
-must also contain the `_key` attribute to identify the document to be replaced. 
+be replaced. `document` must be an object that contains the attributes and values 
+to set. All existing attributes in the stored document are removed from it and
+only the provided attributes are set (excluding system attributes).
```

Comment:
This is the place where `UPDATE` and `REPLACE` differ and should be better stressed, e.g.:
```suggestion
to set. All existing attributes in the stored document are removed from it and
only the provided attributes are set (excluding system attributes). This distinguishes the `REPLACE` operation
from `UPDATE` where the existing attributes are not changed if not mentioned in the `UPDATE` call.
```